product_docs/docs/pgd/5/manual/03-configuring-repositories.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Step 3 - Configuring PGD repositories
+navTitle: Configuring PGD repositories
+deepToC: true
+---
+
+## Configuring PGD repositories
+
+To install and run PGD requires that you configure repositories so that the system can download the appropriate packages.
+
+The following operations should be carried out on each host. For the purposes of this exercise, each host will be a standard data node, but the procedure would be the same for other node types such as witness or subscriber-only nodes.
+
+Obtain your EDB repository token from the [EDB Repos 2.0](https://www.enterprisedb.com/repos-downloads) page.
+
+Set it as en environment variable
```

Comment:
This step seems to be missing an example.

Reply:
Fixed.